docker/release/installer.Dockerfile

```diff
@@ -86,7 +50,7 @@ RUN source /cuda-quantum/scripts/configure_build.sh && \
     ## [<CUDAQuantumAssets]
 
 ## [Self-extracting Archive]
-RUN bash /makeself/makeself.sh --gzip --license /cuda-quantum/LICENSE \
+RUN bash /makeself/makeself.sh --gzip --sha256 --license /cuda-quantum/LICENSE \
```

Comment:
There is a similar `code-block` in `data_center_install.rst`. Does that also need to be updated since you're updating this?

Reply:
It's not necessary but an additional check to validate the archive integrity. I added it to the docs as well.